app/renderer/actions/Session.js

```diff
@@ -310,6 +311,44 @@ export function newSession (caps, attachSessId = null) {
         }
         https = session.server.browserstack.ssl = (parseInt(port, 10) === 443);
         break;
+      case ServerTypes.lambdatest:
+        host = session.server.lambdatest.hostname = process.env.LAMBDATEST_HOST || 'mobile-hub.lambdatest.com';
+        port = session.server.lambdatest.port = process.env.LAMBDATEST_PORT || 443;
+        path = session.server.lambdatest.path = '/wd/hub';
+        isProxyChecked = session.server.advanced.useProxy;
+        username = session.server.lambdatest.username || process.env.LAMBDATEST_USERNAME;
+        if (desiredCapabilities.hasOwnProperty.call(desiredCapabilities, 'lt:options')) {
+          desiredCapabilities['lt:options'].source = 'appiumdesktop';
+          desiredCapabilities['lt:options'].isRealMobile = true;
+          if (isProxyChecked) {
```

Comment:
Can refer to `session.server.advanced.useProxy` directly here and below. Then, `isProxyChecked` could remove as only this block refers to it.